packages/components/inputs/time-input/src/time-input-body.styles.ts

```diff
@@ -8,162 +8,131 @@ import { type TTimeInputProps } from './time-input';
+      ? customProperties.backgroundColorForInputWhenDisabled
       : 'none'};
     border: 0;
-    border-left: 1px solid ${overwrittenVars.borderColorForInput};
-    border-top-right-radius: ${overwrittenVars.borderRadiusForInput};
-    border-bottom-right-radius: ${overwrittenVars.borderRadiusForInput};
-    border-color: ${getClockIconContainerColor(overwrittenVars, props)};
-    color: ${getClockIconContainerFontColor(overwrittenVars, props)};
+    border-left: 1px solid ${customProperties.borderColorForInput};
+    border-top-right-radius: ${customProperties.borderRadiusForInput};
+    border-bottom-right-radius: ${customProperties.borderRadiusForInput};
+    border-color: ${getClockIconContainerColor(props)};
+    color: ${getClockIconContainerFontColor(props)};
     cursor: ${props.isDisabled ? 'not-allowed' : 'default'};
     height: 100%;
     display: flex;
-    padding: ${overwrittenVars.spacingXs};
+    padding: ${customProperties.spacingXs};
     outline: 0;
-    transition: color ${overwrittenVars.transitionStandard},
-      border-color ${overwrittenVars.transitionStandard};
+    transition: color ${customProperties.transitionStandard},
+      border-color ${customProperties.transitionStandard};
     &:hover:not(:disabled):not(:read-only),
     &:focus {
-      border-color: ${overwrittenVars.borderColorForInputWhenFocused};
+      border-color: ${customProperties.borderColorForInputWhenFocused};
     }
   `;
 };
 
-const getInputContainerBorderColor = (
-  vars: TExtendedTheme,
-  props: TTimeInputProps
-) => {
+const getInputContainerBorderColor = (props: TTimeInputProps) => {
   if (props.isDisabled) {
-    return vars.borderColorForInputWhenDisabled;
+    return customProperties.borderColorForInputWhenDisabled;
   }
   if (props.hasError) {
-    return vars.borderColorForInputWhenError;
+    return customProperties.borderColorForInputWhenError;
   }
   if (props.isReadOnly) {
-    return vars.borderColorForInputWhenReadonly;
+    return customProperties.borderColorForInputWhenReadonly;
   }
-  return vars.borderColorForInput;
+  return customProperties.borderColorForInput;
 };
-const getInputContainerFontColor = (
-  vars: TExtendedTheme,
-  props: TTimeInputProps
-) => {
+const getInputContainerFontColor = (props: TTimeInputProps) => {
   if (props.isDisabled) {
-    return vars.fontColorForInputWhenDisabled;
+    return customProperties.fontColorForInputWhenDisabled;
   }
   if (props.hasError) {
-    return vars.fontColorForInputWhenError;
+    return customProperties.fontColorForInputWhenError;
   }
   if (props.isReadOnly) {
-    return vars.fontColorForInputWhenReadonly;
+    return customProperties.fontColorForInputWhenReadonly;
   }
-  return vars.fontColorForInput;
+  return customProperties.fontColorForInput;
 };
-const getInputContainerStyles = (props: TTimeInputProps, theme: Theme) => {
-  const overwrittenVars = {
-    ...customProperties,
-    ...theme,
-  };
```

Comment:
Regarding the change in visual regression tests:
<img width="1005" alt="Screenshot 2022-08-17 at 09 27 04" src="https://user-images.githubusercontent.com/49066275/185061078-0cb3ecbf-1b83-4cdd-84c6-5c3384faf851.png">
it seems to me that the previous visual result was a bug.
`customProperties.backgroundColorForInput` used for `background-color` was not overwritten by the theme.